core/01_core.js

```diff
@@ -179,8 +170,8 @@
     return p;
   }
 
-  function opSync(opName, arg1 = null, arg2 = null) {
-    return unwrapOpResult(opcallSync(opsCache[opName], arg1, arg2));
+  function opSync(opName, arg1, arg2) {
```

Comment:
Just to make sure, both `null` and `undefined` are serialized to the same value by `serde_v8`, right?

Reply:
Yep, though we can restore the old code it's not a material change IMO